adb_graphics/datahandler/grib.py

```diff
@@ -451,6 +451,36 @@ def grid_info(self):
 
         return grid_info
 
+    def supercooled_liquid_water(self, values, **kwargs) -> np.ndarray:
+
+        # pylint: disable=unused-argument
+
+        '''
+        Generates a field of Supercooled Liquid Water
+        '''
+
+        ps = self.values(name='pres', level='sfc') * 100. # convert back to Pa
+        p = self.values(name='pres', level='ua', one_lev=False)
+        t = self.values(name='temp', level='ua', one_lev=False)
+        cmr = self.values(name='clwmr', level='ua', one_lev=False)
+        rmr = self.values(name='rwmr', level='ua', one_lev=False)
+
+        g = 9.81 # gravity
```

Comment:
Why this value for gravity? Should we be using a value that varies with height?

Reply:
according to Curtis, a constant g of 9.81 m/s is fine.